src/migrations/Version20221023115039.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Is it possible to delete all migrations files and recreate them to only have what's relative with User ? ;)

Reply:
Yeah all migration is automatically  deleted when u push in githuh everyone should type symfony console make:migration then doctrine:migration:migrate @MisterAzix

Reply:
Mmmmh sure ? so why is there 2 migrations file here ? 😅

Reply:
Haha tu gères